context/app/static/js/components/Search/SelectedFilter/style.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+import CancelIcon from '@material-ui/icons/CancelRounded';
+
+const StyledCancelIcon = styled(CancelIcon)`
+  height: 1.2rem;
```

Comment:
I think icons are sized using `font-size` in MUI. If this works or has a separate purpose not a big issue.

Reply:
no good reason. fixed.